xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -197,6 +199,15 @@ void CPVRRecording::Serialize(CVariant& value) const
     value["art"]["fanart"] = m_strFanartPath;
 }
 
+void CPVRRecording::ToSortable(SortItem& sortable, Field field) const
+{
+  CSingleLock lock(m_critSection);
```

Comment:
It is correct to introduce the lock here, because m_sizeInBytes gets accessed by multiple threads - at least be the UI thread (calls for example ToSortable) and the PVR Mananger thread (calls for example CPVRRecording::Update).

Thus, all places reading/writing m_sizeInBytes must be mutex protected as well! This part is missing in your PR.